staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -75,14 +79,14 @@ type Event struct {
 	// API object from the request, in JSON format. The RequestObject is recorded as-is in the request
 	// (possibly re-encoded as JSON), prior to version conversion, defaulting, admission or
 	// merging. It is an external versioned object type, and may not be a valid object on its own.
-	// Omitted for non-resource requests.  Only logged at RequestObject Level and higher.
+	// Omitted for non-resource requests.  Only logged at Request Level and higher.
 	// +optional
-	RequestBody string
+	RequestObject runtime.Unknown
 	// API object returned in the response, in JSON. The ResponseObject is recorded after conversion
 	// to the external type, and serialized as JSON.  Omitted for non-resource requests.  Only logged
-	// at ResponseObject Level and higher.
+	// at Response Level.
 	// +optional
-	ResponseBody string
+	ResponseObject runtime.Unknown
```

Comment:
For internal types, runtime.Unknown will be "set" if internal data is nil or not.